Update `.quickstart.yml` and `README` (#26)
fivetran/dbt_linkedin_pages: README.md

| Table | Description |
| :---- | :---- |
| [linkedin_pages__posts](https://fivetran.github.io/dbt_linkedin_pages/#!/model/model.linkedin_pages.linkedin_pages__posts) | Analyzes engagement metrics for LinkedIn posts to understand professional audience reach, interaction patterns, and content performance for your company pages. <br></br>**Example Analytics Questions:**<ul><li>Which posts drive the most engagement (clicks, likes, comments, shares) by content type or topic?</li><li>How does post performance vary by day of week or time of posting?</li><li>What types of content generate the highest reach and impression rates among your professional audience?</li></ul>|

¹ Each Quickstart transformation job run materializes these models if all components of this data model are enabled. This count includes all staging, intermediate, and final models materialized as `view`, `table`, or `incremental`.

## Prerequisites
To use this dbt package, you must have the following:

- At least one Fivetran Linkedin Pages connection syncing data into your destination.
- A BigQuery, Snowflake, Redshift, PostgreSQL, or Databricks destination.

> ## How do I use the dbt package?

Reviewer: ```suggestion
<!--section-end-->

## How do I use the dbt package?
```

    <a alt="PRs">
        <img src="https://img.shields.io/badge/Contributions-welcome-blueviolet" /></a>
    <a alt="Fivetran Quickstart Compatible"
        href="https://fivetran.com/docs/transformations/data-models/quickstart-management#quickstartmanagement">
        <img src="https://img.shields.io/badge/Fivetran_Quickstart_Compatible%3F-yes-green.svg" /></a>
</p>

This dbt package transforms data from Fivetran's Linkedin Pages connector into analytics-ready tables.

## Resources

- Number of materialized models¹: 15
- Connector documentation

>   - [Linkedin Pages connector documentation](https://fivetran.com/docs/connectors/applications/linkedin-pages)

Reviewer: ```suggestion
  - [Linkedin Pages connector documentation](https://fivetran.com/docs/connectors/applications/linkedin-company-pages)
```

>   - [Linkedin Pages ERD](https://fivetran.com/docs/connectors/applications/linkedin-pages#schemainformation)

Reviewer: ```suggestion
  - [Linkedin Pages ERD](https://docs.google.com/presentation/d/1PbexBiOTxplv7TlmNZ7uNo8EDT2Lr5g0Ys_ealh4qzI/edit?slide=id.g311502b468_5_443#slide=id.g311502b468_5_443)
```

  - [GitHub repository](https://github.com/fivetran/dbt_linkedin_pages)
  - [dbt Docs](https://fivetran.github.io/dbt_linkedin_pages/#!/overview)
  - [DAG](https://fivetran.github.io/dbt_linkedin_pages/#!/overview?g_v=1)
  - [Changelog](https://github.com/fivetran/dbt_linkedin_pages/blob/main/CHANGELOG.md)

## What does this dbt package do?
This package enables you to transform core social media object tables into analytics-ready models and generate comprehensive data dictionaries. It creates enriched models with metrics focused on LinkedIn post performance.

The main focus of the package is to transform the core social media object tables into analytics-ready models that can be easily unioned in to other social media platform packages to get a single view. This is especially easy using our [Social Media Reporting package](https://github.com/fivetran/dbt_social_media_reporting).

### Output schema
Final output tables are generated in the following target schema:

```
<your_database>.<connector/schema_name>_linkedin_pages
```

### Final output tables

By default, this package materializes the following final tables:

| Table | Description |
| :---- | :---- |

> | [linkedin_pages__posts](https://github.com/fivetran/dbt_linkedin_pages/blob/main/models/linkedin_pages__posts.sql) | Analyzes engagement metrics for LinkedIn posts to understand professional audience reach, interaction patterns, and content performance for your company pages. <br></br>**Example Analytics Questions:**<ul><li>Which posts drive the most engagement (clicks, likes, comments, shares) by content type or topic?</li><li>How does post performance vary by day of week or time of posting?</li><li>What types of content generate the highest reach and impression rates among your professional audience?</li></ul>|

Reviewer: ```suggestion
| [linkedin_pages__posts](https://fivetran.github.io/dbt_linkedin_pages/#!/model/model.linkedin_pages.linkedin_pages__posts) | Analyzes engagement metrics for LinkedIn posts to understand professional audience reach, interaction patterns, and content performance for your company pages. <br></br>**Example Analytics Questions:**<ul><li>Which posts drive the most engagement (clicks, likes, comments, shares) by content type or topic?</li><li>How does post performance vary by day of week or time of posting?</li><li>What types of content generate the highest reach and impression rates among your professional audience?</li></ul>|
```

¹ Each Quickstart transformation job run materializes these models if all components of this data model are enabled. This count includes all staging, intermediate, and final models materialized as `view`, `table`, or `incremental`.

> ---

Reviewer: ```suggestion
```

## Prerequisites
To use this dbt package, you must have the following:

- At least one Fivetran Linkedin Pages connection syncing data into your destination.
- A BigQuery, Snowflake, Redshift, PostgreSQL, or Databricks destination.

> 

Reviewer: <!--section-end-->

## How do I use the dbt package?
You can either add this dbt package in the Fivetran dashboard or import it into your dbt project:

- To add the package in the Fivetran dashboard, follow our [Quickstart guide](https://fivetran.com/docs/transformations/data-models/quickstart-management).
- To add the package to your dbt project, follow the setup instructions in the dbt package's [README file](https://github.com/fivetran/dbt_linkedin_pages/blob/main/README.md#how-do-i-use-the-dbt-package) to use this package.

> ### Installing the Package

Reviewer: ```suggestion
### Install the Package
```
